docs/docs/deployment/configuration/importing-and-exporting.md

```diff
@@ -141,6 +141,13 @@ make sure whichever role you are using to run you container has access to read f
 Alternatively, you can provide the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables to refer to an
 IAM user that has access to the S3 bucket.
 
+> Using localstack to achieve local/test exports with s3 can be done using
```

Comment:
Can we add this just as a regular heading - we don't use Quote formatting anywhere in the docs - thanks!